README.md

```diff
@@ -1,73 +1,108 @@
+
+Getting Started
+---------------
+### Prerequisites
+You will need the following tools:
+- GNU make
+- [clang](http://clang.llvm.org/)
+- [GNU bison](https://www.gnu.org/software/bison/) and [flex](http://flex.sourceforge.net/)
+- [wine](https://www.winehq.org/), to invoke the NVIDIA Cg compiler
+- [lld](http://lld.llvm.org/) or the [MinGW](http://www.mingw.org/) flavor of GNU ld.
+- [Git](http://git-scm.com/)
 
-- 3D graphics! nxdk includes and expands on pbkit, a library for interfacing with the Xbox GPU.
-
-nxdk is currently in early stages of development (read: it was hacked together over a weekend).
-
-How?
-----
-
-You need:
+#### OS X
+On OSX with [Homebrew](http://brew.sh/), this should do the job (XCode ships with make and bison and flex):
 
-- make
+    brew install wine llvm --with-lld --with-clang
 
-- A recent version of [clang](http://clang.llvm.org/) and [lld](http://lld.llvm.org/)
+#### Linux (Ubuntu)
+On Ubuntu, the requried packages can be downloaded from the standard repositories:
 
-- [GNU bison](https://www.gnu.org/software/bison/) and [flex](http://flex.sourceforge.net/)
+    sudo apt-get install build-essential flex bison g++ clang wine binutils-mingw-w64 git
 
-- *Temporarilly because I'm lazy*, [wine](https://www.winehq.org/) to invoke the NVIDIA Cg compiler
+### Download NXDK
+    git clone https://github.com/xqemu/nxdk.git
+    cd nxdk
 
-On OSX with [Homebrew](http://brew.sh/), this might do the job (XCode ships with make and bison and flex):
+NXDK comes with a set of tools necessary for building. Build them with:
 
-    brew install wine llvm --with-lld
+    make tools
 
-Now you can compile your application and nxdk:
+### Build Samples
+To build the 0ldskoo1 sample, you can run:
 
+    cd samples/0ldskoo1
     make
 
-Credits
--------
+This will generate a single executable, default.xbe, in the bin/ directory which
+can be executed on your Xbox (or XQEMU emulator).
 
-- [OpenXDK](http://openxdk.maturion.de/) is inspiration for nxdk, and large parts are reused.
-    - License: GPLv2
-- Large parts of [pbkit](http://forums.xbox-scene.com/index.php?/topic/573524-pbkit) by openxdkman are included, with modifications.
-    - License: Academic Free License
-- A very barebones libc is included based on [lib43](https://github.com/lunixbochs/lib43)
-    - License: MIT
-- vp20compiler is based on nvvertparse.c from [Mesa](http://www.mesa3d.org/)
-    - License: MIT
-- fp20compiler is based on nvparse from the [NVIDIA SDK 9.52](https://www.nvidia.com/object/sdk-9.html).
-- A (windows) copy of the NVIDIA Cg compiler 1.3 from the NVIDIA SDK is bundled.
+### Generate XISO
+To generate an ISO file that can be burned to a disc, or passed to the XQEMU
+emulator via the `-drive index=1,media=cdrom,file=/path/to/your.iso` parameter,
+define the `GEN_XISO` variable with the name of the ISO to be created in your
+project Makefile. For example:
 
-Code Overview
--------------
+    GEN_XISO=$(XBE_TITLE).iso
```

Comment:
I don't really think we should generate ISOs by default. It's pretty simple though, just add GEN_XISO=filename.iso to your Makefile. Or you can do it when you run make. e.g. `make GEN_XISO=test.iso`.